resources/Materials/TestSuite/stdlib/upgrade/1_36_to_1_37.mtlx

```diff
@@ -117,10 +117,6 @@ Upgrade path test from 1.36 to 1.37
     <input name="in" type="vector3" value="1.0, 0.5, 0.0"/>
     <input name="mat" type="matrix33" value="0.5, 0.0, 0.0, 0.0, 2.0, 0.0, 0.0, 0.0, 0.5"/>
   </transformnormal>
-  <transformnormal name="normal_transform34" type="vector4">
```

Comment:
Curious as to why this case is no longer tested ?

Reply:
With the new validation logic, what I discovered is that we never truly supported the upgrade of this particular legacy node in 1.36, since this upgrade path would require a change of node signature (not just a change of name).  If needed, we can add support for that upgrade pathway, but I've removed it from our test as it's not currently a supported feature.